src/data-types/constants.ts

```diff
@@ -83,7 +83,7 @@ export const userLedActionList: EventsEnum[] = [
   EventsEnum.AUTH_PASSWORD_RESET_SUCCESSFUL_FOR_TEST_CLIENT,
 ];
 
-export const nonInterventionsCodes = new Set(['90', '91', '92', '93']);
+export const nonInterventionsCodes = new Set(['90', '91', '92', '93', '94', '95']);
```

Comment:
could we do this dynamically based on edges that do not have an `interventionName`?